src/main/thrift/parquet.thrift

```diff
@@ -162,19 +162,48 @@ enum ConvertedType {
   BSON = 20;
 
   /**
+   * @Deprecated: use INTERVAL_YEAR_MONTH or INTERVAL_DAY_TIME
+   * since the SQL standard defines either YEAR_MONTH or DAY_TIME unit.
+   * This is deprecated in favor of those 2 types
```

Comment:
SQL 92 Spec:

         4.5.2  Intervals
         There are two classes of intervals. One class, called year-month
         intervals, has an express or implied datetime precision that in-
         cludes no fields other than YEAR and MONTH, though not both are
         required. The other class, called day-time intervals, has an ex-
         press or implied interval precision that can include any fields
         other than YEAR or MONTH.

Further:

         Operations involving items of
         type interval require that the interval items be mutually compara-
         ble.
         ...
         Year-month intervals are mutually comparable only with other year-
         month intervals
         ...
         Day-time intervals are mutually comparable only with other day-
         time intervals.